snuba/query/conditions.py

```diff
@@ -66,6 +68,19 @@ def __is_set_condition(exp: Expression, operator: str) -> bool:
     return False
 
 
+def __set_condition_pattern(
+    lhs: Pattern[Expression], operator: str
+) -> FunctionCallPattern:
+    return FunctionCallPattern(
+        None,
+        String(operator),
+        (
+            Param("lhs", lhs),
+            Param("tuple", FunctionCallPattern(None, String("tuple"), None)),
+        ),
+    )
+
+
 def in_condition(
```

Comment:
I think at a point we may remove the old helper methods (is_in_condition, is_binary_condition) and just replace all of them with the Patterns.